KomodoPlatform/komodo-wallet-desktop
# Fix and improve tendermint withdraws

This PR adds a warning for Tendermint withdrawals, and adjusts the layout of the send form to include optional IBC channel ID input.

To Test:
- Login and go to send some Tendermint coins
- See warnings about IBC transfers to CEX
- Send tendermint coin with non-IBC transfer (same chain/address format)
- Send tendermint coin with IBC transfer but without custom IBC channel
- Send tendermint coin with IBC transfer and with custom IBC channel
- Confirm link to IBC channel info working
- Test transactions with some other coins to confirm no regression (with/without custom fees & memos)
